chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTest.java

```diff
@@ -0,0 +1,145 @@
+package com.griddynamics.jagger.user.test.configurations;
+
+/**
+ * The user
+ *
+ * @author asokol
+ *         created 10/18/16
+ */
+public class JTest {
+
+
+    private String id;
+    private JTestDescription jTestDescription;
+    private long requestsPerSecond;
+    private long maxLoadThreads;
+    private long warmUpTimeInSeconds;
+    private TerminationType terminationType;
+    private long durationInSeconds;
+    private long iterationCount;
+    private long maxDurationInSeconds;
+
+
+    public enum TerminationType {
```

Comment:
1. Yes, that would be nice.
2. Also, I thought Termination would be a separate entity?